README.md

```diff
@@ -1,2 +1,29 @@
 # devtools
 Development tools for SlamData
+
+## Instructions
+How to get up and running with `merge`.
+
+0. Before using merge you must first approve the pull request you have been assinged. 
+You can do this under the `Files Changed` tab of Github's PR page. Then click the green `Review Changes` button on the left.
+
```

Comment:
Somewhere around here, you also want to ensure the PR has a “version:” label on it.